Latex Prepocessing for PDF Conversion

dfeldman987 opened the pull request.

dfeldman-osii committed:
Add pdf preprocessor to replace latex with svgs

dfeldman-osii committed:
Refactor and cleanup latex processing

dfeldman987 committed:
remove monitor code in latex

dfeldman987 committed:
Undo spacing

dfeldman987 committed:
Undo spacing from parent

Joxli commented on asciidoctor-editor-plugin/src/main/java-eclipse/de/jcup/asciidoctoreditor/LatexPDFPreprocessor.java:
```diff
@@ -0,0 +1,347 @@
+        		
+        		BufferedReader reader = new BufferedReader(
+    			    new InputStreamReader(process.getInputStream())
+    			);
```
Does any of these need to be closed?

Joxli reviewed (approved):
Looks good. The indentations seems off. Other than that, only small questions.

dfeldman987 committed:
Moved latex processing into its own projects to match asciidoctor editor plugin setup

dfeldman987 committed:
Fix spacing, update JS script

dfeldman987 committed:
try to fix spacing

dfeldman987 committed:
Update try catch formatting

tviegut commented on asciidoctor-converter-pdf-latex/build.properties:
Just double check the additional source code directory (src/main/java) to confirm.

dfeldman987 committed:
Fix security check, add refresh

dfeldman987 committed:
Gradle and project fixes

markdown to asciidoc was update to 2.0 in gradle but not in the build.
Included required project files to detect new project
